policy/modules/system/systemd.if

```diff
@@ -61,6 +57,7 @@ template(`systemd_role_template',`
 	allow $1_systemd_t $3:process { setsched rlimitinh signal_perms };
 	corecmd_shell_domtrans($1_systemd_t, $3)
 	corecmd_bin_domtrans($1_systemd_t, $3)
+	corecmd_shell_entry_type($1_systemd_t)
```

Comment:
When is this needed?

Reply:
Doesn't seem to be needed now so I'll remove it.